diagnostic.md

```diff
@@ -27,7 +27,11 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+$base-font-size: ($font-size * 1.3);
+
+h1 {
+  font-size: $base-font-size;
+}
 ```
```

Comment:
This will get you the correct `h1` size, but you'd want to keep the `base-font-size` untouched, and perform the math on the `h1`'s font size like so:

```CSS
h1 {
  font-size: $base-font-size * 1.3;
}
```